tests/unit/hive_metastore/test_catalog_schema.py

```diff
@@ -10,126 +10,168 @@
 from databricks.sdk.service.catalog import CatalogInfo, ExternalLocationInfo, SchemaInfo
 
 from databricks.labs.ucx.hive_metastore.catalog_schema import CatalogSchema
-from databricks.labs.ucx.hive_metastore.grants import PrincipalACL, Grant, GrantsCrawler
+from databricks.labs.ucx.hive_metastore.grants import Grant, MigrateGrants
 from databricks.labs.ucx.hive_metastore.mapping import TableMapping
-
-
-def prepare_test(ws, backend: MockBackend | None = None) -> CatalogSchema:
-    ws.catalogs.list.return_value = [CatalogInfo(name="catalog1")]
+from databricks.labs.ucx.workspace_access.groups import GroupManager
+
+
+def prepare_test(ws, backend: MockBackend | None = None) -> CatalogSchema:  # pylint: disable=too-complex
+    """Prepare tests with the following setup:
```

Comment:
Did a clean up of the `prepare_tests` to verify it is testing what I expect it does